src/test.cpp

```diff
@@ -88,10 +88,9 @@ int main(int argc, char *argv[])
 		auto db2 = display->createDisplayBuffer(300, 200, 32);
 		auto fb2 = display->createFrameBuffer(db2, 300, 200, 1);
 
-		for(size_t i = 0; i < db1->getSize() / 4; i++)
-		{
-			(reinterpret_cast<uint32_t*>(db1->getBuffer()))[i] = 0xFFAACC;
-		}
+		uint32_t *uiBegin = static_cast<uint32_t * >(db1->getBuffer());
```

Comment:
AFAIK, we don't need space after `*`.

We are trying to have codebase with consistent style. So, if you have a doubts on how to format specific piece of code - try looking for the similar pieces in other files.

This also true for variable naming and for commit descriptions.